### DISCUSSION TITLE
Accessing sketch title within sketch

### DISCUSSION THREAD

**Discussion by coffeeintothms** [processing/p5.js-web-editor#2670] [General]
### Increasing Access

Unsure

### Feature request details

It might be useful to make the sketch title available as a variable within the sketch. Often I want to save a png or gif from the editor, and it would be nice to simply run something like
```
saveGif(`${title}.gif`,5)
```

**Comment by welcome** [comment]
Welcome! 👋 Thanks for opening your first issue here! And to ensure the community is able to respond to your issue, be sure to follow the issue template if you haven't already.
